### ISSUE TITLE
[Load Testing] Add .NET6 Benchmark Performance Apps

### ISSUE THREAD

**Issue by satvu** [Azure/azure-functions-host#8600]
Add .NET6 Benchmark apps to the load-testing repo. The app and terraform files are ready but need to be properly integrated into the newly refactored repository and directory structure.

- [x] Merge BenchmarkApp solution
- [x] Add a pipeline for Benchmark Apps (v3 and v4)
- [x] Merge Benchmark App terraform scripts into refactored directory

**Comment by fabiocav** [comment]
Marking as blocked by work from @vrdmr's team that is currently in progress.  Will pick this back up when we can move forward.

**Comment by vrdmr** [comment]
@fabiocav - you can close this issue now. @arroyc finished adding the .net6 apps and throughput pipeline is up and running for them.

[Pull request 90: Add webapp bencmark project net6 - Repos (visualstudio.com)](https://azfunc.visualstudio.com/Azure%20Functions/_git/FunctionAppsForLoaderTests/pullrequest/90?_a=files)

**Comment by satvu** [comment]
@vrdmr @arroyc Are the webapps included in that pipeline? I see that the new dotnet pipelines pull from "_TerraformTemplates/dotnet-isolated/*_" or "_TerraformTemplates/dotnet/*_" but the webapp templates are still in the root _/TerraformTemplates_ folder in the folders starting with "wapf". 

I have a [PR](https://azfunc.visualstudio.com/Azure%20Functions/_git/FunctionAppsForLoaderTests/pullrequest/87) from before the refactor to deploy new benchmark webapps, but unclear on how to proceed with the new folder structure.

**Comment by satvu** [comment]
Synced with @arroyc and he walked me through the new webapps directory with the dotnet folder inside for us to add the necessary templates - will continue with this task now. We are also now testing WestUS only, so will likely just make a new PR and abandon the old one.

**Comment by fabiocav** [comment]
This is being iterated on. Currently in PR.

**Comment by satvu** [comment]
Last task (3/3) is awaiting final review [here](https://azfunc.visualstudio.com/Azure%20Functions/_git/FunctionAppsForLoaderTests/pullrequest/221).

**Comment by fabiocav** [comment]
Keeping this in 131.

@arroyc can you please assist so we can close on this item?

**Comment by satvu** [comment]
closing as all tasks have been completed
